apps/health_campaign_field_worker_app/lib/blocs/facility/facility.dart

```diff
@@ -40,6 +40,12 @@ class FacilityBloc extends Bloc<FacilityEvent, FacilityState> {
           version: 1,
           fields: [const AdditionalField('type', 'ProvisionalWarehouse')],
         ),
+        auditDetails: AuditDetails(
+          createdBy: 'createdBy',
```

Comment:
@vishal-egov  Pls add the createdBy and updatedBy

Reply:
@naveen-egov this is a dummy warehouse we areadding so there is no createdby or lastmodified for this facility, had to add it becase it i mandatory

Reply:
add the user id  instead of empty string